src/olympia/addons/serializers.py

```diff
@@ -775,9 +758,10 @@ class AddonBasketSyncSerializer(AddonSerializerWithUnlistedData):
     # We want to send all authors to basket, not just listed ones, and have
     # the full basket-specific serialization.
     authors = UserProfileBasketSyncSerializer(many=True)
-    name = serializers.SerializerMethodField()
-    latest_unlisted_version = VersionBasketSerializer()
     current_version = VersionBasketSerializer()
+    is_recommended = serializers.SerializerMethodField()
```

Comment:
Why does this need to be added now?

Reply:
it's been removed above from super class `AddonSerializer`